tests/unit/topology/test_graph.py

```diff
@@ -42,7 +42,7 @@ def visualize_graphs():
     """Render graphs when running this file directly."""
     ui.load_default_particles()
     topology = create_dummy_topology()
-    graphs = test_initialize_graph(topology, None)
+    graphs = test_initialize_graph(topology, io.load_pdg())
```

Comment:
Isn't particle_database a fixture? So then that second argument would not be needed

Reply:
Yeah but a fixture is only loaded when running through `pytest`. This is only there when you directly run the `__main__`, it's a trick to visualise the topology if you do have `graphviz`.

Reply:
Ok, but is that not using the fixture then in case of running with pytest?

Reply:
The function `visualize_graphs` is not called through `pytest` at all, because it doesn't start or end in `test`.

Reply:
Note also that this is the reason for the slightly weird construct here
https://github.com/ComPWA/expertsystem/blob/f63f8a6e958cfa74678f1e10eef4d9922c84ffae/tests/unit/topology/test_graph.py#L44-L58

Reply:
Ah I completely missed that. Ok, so then depending on the purpose of this code I would either promote it to a test or change it into some example code/jupyter notebook (can be handled in a separate PR). However the purpose of the code is not quite clear to me

Reply:
It's mainly just a way to (if you have the right setup) quickly check what the dummy topology looks like